### PR TITLE
WIP: move search bar into the header

### PR DESCRIPTION
**Pull request by fuentesloic**

### CODE REVIEW

File: static/js/search.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+url = window.location.href
```
Comment (vinyll) [inline static/js/search.js]: You may be interested into the following magic:

```js
const query = window.location.search.slice(1).split('&').map(kv => kv.split('=')).map(arr => [arr[0], unescape(arr[1])])
const q = query.get('q')

Array.from(document.querySelectorAll('#search input')).forEach(input => {
  input.value = q
})
```

**Comment by fuentesloic** [comment]
capture d ecran 2017-08-14 a 16 14 52-fullpage

**Comment by vinyll** [comment]
I'd rather imagine something more discreet and incrusted, such as:

search example

@AymericFaivre / @newick: opinions or suggestions?

**Comment by AymericFaivre** [comment]
Hey !

Yep, maybe just put the magnifying glass on the top left, and deploy the form on click. Maybe with just the glass in green. (like on medium, I like this kind of interaction)

Or put the first or second @vinyll's example to the top left and skin it by putting the magnifying glass in green and the border radius at max.

**Comment by newick** [comment]
I agree with @AymericFaivre for the interaction and the rest :D

**Comment by fuentesloic** [comment]
following the issue #39 
So for now, not enought content

**Comment by vinyll** [comment]
According to https://github.com/neomad-team/neomad.org/issues/39#issuecomment-323245455, this PR is being postponed.

**Pull request closed without merging**